crystal/release-build.yaml

```diff
@@ -0,0 +1,94 @@
+%YAML 1.1
+# ROS buildfarm release-build file
+---
+abi_incompatibility_assumed: true
+
+build_environment_variables:
+  RTI_NC_LICENSE_ACCEPTED: 'yes'
+
+jenkins_binary_job_priority: 82
+jenkins_binary_job_timeout: 120
+jenkins_source_job_priority: 78
```

Comment:
The job priorities shouldn't be the same as the ones for Bouncy. All amd64 builds should be prioritized over aarch64 builds and newer ROS distros should be prioritized over older ROS distros.

Reply:
> ll amd64 builds should be prioritized over aarch64 builds

Since the ROS 2 builfdarm runs AMD and ARM jobs on distinct hosts the permissions for these jobs was "tied" for bouncy and ardent. I would be inclined to keep the same policy for crystal.